desktop/mac/release-notes.md

```diff
@@ -12,14 +12,34 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.0.1
+2021-09-09
+
+> Download Docker Desktop
+>
+> [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
+> [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
+
+### Upgrades
+
+- [Compose V2 RC3](https://github.com/docker/compose-cli/releases/tag/v2.0.0-rc.3)
+  - Compose v2 is now hosted on github.com/docker/compose.
+  - Fixed go panic on downscale using `compose up --scale`.
+  - Fixed  a race condition in `compose run --rm` while capturing exit code.
+
+### Bug fixes and minor changes
+
+- Fixed a bug where copy-paste was not available in the Dashboard.
 
 ## Docker Desktop 4.0.0
 2021-08-31
 
 > Download Docker Desktop
 >
-> [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
-> [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
+> {%- include eula.md -%}
+>
+> [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/67817/Docker.dmg){: .accept-eula } |
+> [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/67817/Docker.dmg){: .accept-eula }
```

Comment:
I think the accept-eula classes are not needed for v4.0 and up (because docker desktop itself will ask the user to accept)